### PR TITLE
feature/SBBINCL-30

### PR DESCRIPTION
**Pull request by fleuryj**
The missing elements for SBBTabView are added:
- Landscape mode
- VoiceOver
- Translation
- Test
- Documentation

### CODE REVIEW

File: SBBDesignSystemMobileSwiftUI/Views/SBBTabView/TabBarView/TabButtonView.swift
Comment (fleuryj) [inline SBBDesignSystemMobileSwiftUI/Views/SBBTabView/TabBarView/TabButtonView.swift:123]: This line should remove the Button trait, i.e. the VoiceOver should not say 'Button' any more. However, it still does say it.
As of now it says:
  - for the selected tab: Selected. Tab title. Tab. Button. 1/5.
  - for the other ones: Tab title. Tab. Button. 2/5.

**Review by fvessaz — COMMENTED** [review]
Looks good to me.
Can be merged after merging the enhancements PR and reviewing the documentation.

**Review by fvessaz — APPROVED** [review]

**Pull request merged**
